Depth_First_Search/Depth_First_Search.dart

```diff
@@ -0,0 +1,74 @@
+// Required library to use inbuilt queue in LIFO manner in Dart
+import 'dart:collection';
+// Required library to use standard output facility of dart
+import 'dart:io';
+
```

Comment:
Add space around operators throughout the file

Reply:
Changes made